### PR TITLE
Fix alignment problem

### PR DESCRIPTION
**Pull request by DCDBionicman**
Fix alignment problem on scrolling
изображение

### CODE REVIEW

File: TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java [hunk 1]
```diff
@@ -198,8 +198,8 @@ public boolean dispatchTouchEvent(MotionEvent ev) {
 
     private <T extends View> void dispatchEventForView(MotionEvent ev, T genericView) {
         //genericView.getLocationInWindow(location);
-        int height = timelineRecyclerView.getHeight();
-        int width = channelsRecyclerView.getWidth();
+        int height = genericView.getHeight();
```
Comment (Duna) [inline TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java:201]: by doing so, is the programs clicked correctly when click near the margins? Have to take into account the left width of channels logo and height of top timeline, then dispatch the recalculated event position to the subviews. This commit is not fixing the sporadic issue with vertical alignment

**Review by Duna — CHANGES_REQUESTED** [review]

File: TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java (hunk 1 above)
Comment (Duna) [reply TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java:201]: The fix will imply the LayoutManager and my Observer class. Anyway this is very rare issue, if using this lib on TV you will not have this issue. If using on phone and scroll very fast then you can reproduce the issue. Please report the issue on lib page.

Comment (DCDBionicman) [reply TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java:201]: Oh, I see. Thank you!

Comment (Duna) [reply TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java:201]: Did you find a solution?

Comment (DCDBionicman) [reply TwoWayRecyclerView/genericepg/src/main/java/genericepg/duna/project/view/EPGView.java:201]: I realized that the problem is not on the surface and have not yet dived into details.

**Comment by Duna** [comment]
Fixed on main branch

**Pull request closed without merging**